WordPress/Classes/ViewRelated/Post/PostCardStatusViewModel.swift

```diff
@@ -204,7 +204,7 @@ class PostCardStatusViewModel: NSObject, AbstractPostMenuViewModel {
             buttons.append(.moveToDraft)
         }
 
-        if post.status == .publish || post.status == .draft {
+        if post.status == .publish || post.status == .draft || post.status == .pending {
```

Comment:
I only made these changes to the posts view model, because there's no such thing as pending pages - contributors can't create pages (ref: https://wordpress.com/support/invite-people/user-roles/)